### PR TITLE
Date time format

### PR DESCRIPTION
**Pull request by AbijithS-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes

Changed date and time format as configurabale from the env file
# How has the change been tested?

# Screenshots (if applicable)

### CODE REVIEW

File: forms-flow-web-root-config/docker-compose.yml [hunk 1]
```diff
@@ -43,6 +43,8 @@ services:
       - REACT_APP_ENABLE_DASHBOARDS_MODULE=${ENABLE_DASHBOARDS_MODULE:-true}
       - REACT_APP_ENABLE_PROCESSES_MODULE=${ENABLE_PROCESSES_MODULE:-true}
       - REACT_APP_ENABLE_APPLICATIONS_MODULE=${ENABLE_APPLICATIONS_MODULE:-true}
+      - REACT_APP_DATE_FORMAT=${DATE_FORMAT}
```
Comment (abhilash-aot) [inline forms-flow-web-root-config/docker-compose.yml:46]: Add default values

File: deployment/docker/sample.env [hunk 2]
```diff
@@ -199,6 +199,10 @@ CUSTOM_SUBMISSION_URL=http://{your-ip-address}:{port}
 #ENABLE_PROCESSES_MODULE=true
 #ENABLE_APPLICATIONS_MODULE=true
 
+# Add date and time format default will be DD-MM-YY and hh:mm:ss A
+#DATE_FORMAT=
```
[the remark this answers is not in the document]
Comment (AbijithS-aot) [reply deployment/docker/sample.env:203]: default values are already taking from the constants and in the comment I have mentioned the default values

**Comment by AbijithS-aot** [comment]
For test cases in web helperservice will be available only in the run time so it cannot be expected in the screen during test run which will get undefined since it is available only in the runtime. So removing those cases from the web

File: forms-flow-web-root-config/docker-compose.yml (hunk 1 above)
Comment (AbijithS-aot) [reply forms-flow-web-root-config/docker-compose.yml:46]: added

**Review by abilpraju-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**